content/en/docs/tasks/configure-pod-container/migrate-from-psp.md

```diff
@@ -19,30 +19,552 @@ admission controller. This can be done effectively using a combination of dry-ru
+   level](#verify-pss-level), you can test out both the Baseline and Restricted levels to see
+   whether they are sufficiently permissive for existing workloads, and chose the least-privileged
+   valid level.
+
+{{< caution >}}
+Options 2 & 3 above are based on _existing_ pods, and may miss workloads that aren't currently
+running, such as CronJobs, scale-to-zero workloads, or other workloads that haven't rolled out.
+{{< /caution >}}
+
+### 3.b. Verify the Pod Security level {#verify-pss-level}
+
+Once you have selected a Pod Security level for the namespace (or if you're trying several), it's a
+good idea to test it out first (you can skip this step if using the Privileged level). Pod Security
+includes several tools to help test and safely roll out profiles.
+
+First, you can dry-run the policy, which will evaluate pods currently running in the namespace
+against the applied policy:
+```sh
+# $LEVEL is the level to dry-run, either "baseline" or "restricted".
+kubectl label --dry-run=server --overwrite ns $NAMESPACE pod-security.kubernetes.io/enforce=$LEVEL
+```
+This command will return a warning for any _existing_ pods that are not valid under the proposed
+level.
+
+The second option is better for catching workloads that are not currently running: audit mode. When
+running under audit-mode (as opposed to enforcing), pods that violate the policy level are recorded
+in the audit logs, which can be reviewed later after some soak time. Warning mode works similarly,
+but returns the warning to the user immediately. You can set the audit level on a namespace with
+this command:
+```sh
+kubectl label --overwrite ns $NAMESPACE pod-security.kubernetes.io/audit=$LEVEL
+```
+
+If either of these approaches yield unexpected violations, you will need to either update the
+violating workloads to meet the policy requirements, or relax the namespace Pod Security level.
+
+### 3.c. Enforce the Pod Security level
+
+When you are satisfied that the chosen level can safely be enforced on the namespace, you can update
+the namespace to enforce the desired level:
+
+```sh
+kubectl label --overwrite ns $NAMESPACE pod-security.kubernetes.io/enforce=$LEVEL
+```
+
+### 3.d. "Disable" PodSecurityPolicy
+
+Finally, you can effectively "disable" PodSecurityPolicy at the namespace level by binding the fully
+{{< example file="policy/privileged-psp.yaml" >}}privileged PSP{{< /example >}} to all service
+accounts in the namespace.
+
+```sh
+# The following cluster-scoped commands are only needed once.
+kubectl apply -f privileged-psp.yaml
+kubectl create clusterrole privileged-psp --verb use --resource podsecuritypolicies.policy --resource-name privileged
+
+# Per-namespace disable
+kubectl create -n $NAMESPACE rolebinding disable-psp --clusterrole privileged-psp --group system:serviceaccounts:$NAMESPACE
+```
+
+Since the privileged service account is non-mutating, and the PSP admission controller always
```

Comment:
This is the first time we are talking about "privileged service account" ... why service account is relevant here?